pkg/cloudprovider/providers/cloudstack/cloudstack.go

```diff
@@ -20,9 +20,11 @@ import (
 	"fmt"
 	"io"
 
+	gcfg "gopkg.in/gcfg.v1"
```

Comment:
this should be in a section with the github.com/* imports; the k8s.io imports should be alone in the 3rd grouping.